src/util/number.ts

```diff
@@ -279,6 +279,63 @@ export function getPercentWithPrecision(valueList: number[], idx: number, precis
     return seats[idx] / digits;
 }
 
+/**
+ * Get a data of given precision, assuring the sum of percentages
+ * in valueList is 1.
+ * The largest remainer method is used.
+ * https://en.wikipedia.org/wiki/Largest_remainder_method
+ *
+ * @param valueList a list of all data
+ * @param precision integer number showing digits of precision
+ * @return {Array<number>} percent ranging from 0 to 100
+ */
+export function getPercentSeats(valueList: number[], precision: number): number[] {
```

Comment:
Seems getPercentSeats shares lots of logic with getPercentWithPrecision. Could we reuse these code?